.github/workflows/publish-img.yaml

```diff
@@ -39,6 +40,10 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Run pre-build script
+        if: github.repository_owner == 'kubevirt'
+        run: ./passt/vendor-kubevirt.sh
```

Comment:
I though we were going to use the pre-build passt binding CNI from kubevirt/kubevirt

Reply:
But don't we need changes to the CNI to be able to report the interface's IPs ?

Reply:
The upstream pre-build is only released when there is a new kubevirt release,
and it is only a binary, not a container image.
Here we vendor, allow to control which is the hash, and even add our own logic (preferably we will just able to do it on kubevirt itself and not change it after vendoring).
This PR also added a Dockerfile that allows building the CNI binary and wrapping it as an image,
as close as we already have on non upstream code.

Reply:
@maiqueb we don't we will build different CNI binary for our purposes

@oshoval you should use the CNI binary released by kubevirt/kubevirt, right now we don't need to fork and modify since we are going to chain CNIs to do the cni reporting trick.

Reply:
wdyt @maiqueb ?
imho we can just compile it as it is now, unless you both prefer to use the current binary,
one of the reasons it that it will allow to easily test non yet released versions, otherwise how would you test unreleased binaries ?
you would need to build and hack the Dockerfile etc,
with the current approach it is much easier, as it is "dev ready".
Also note it affects only upstream, which also gives us some more flexibility to choose whatever we want.

Reply:
I think passt-cni-binding is at nightlies too 
```
LATEST=$(curl -L https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/latest)
$ kubectl apply -f https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/${LATEST}/kubevirt-operator.yaml
$ kubectl apply -f https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/${LATEST}/kubevirt-cr.yaml
```

If needed we can put a mechanism to use nightlies.

Reply:
ok atm wont add it